### ISSUE TITLE
Compatibilitywith v11

### ISSUE THREAD

**Issue by parvusdomus** [Foundry-VTT-Lex-Arcana/foundryvtt-lexarcana#93]
Did an initial check and found only a deprecation warning with the custos type but roll dialogs were not working most of the time without any warning or error message. I suspect the latter is due to the v11 being in a very early state and not a fault of the code. Will check again in coming versions. This issue is here so I don't forget to do so.

**Comment by parvusdomus** [comment]
There's still problems with the createdialog function. According to the foundry devels it's a problem with the HTML sanitization process. They'll have to look at it fo fixing. I'll still have to wait for next version and see if it's fixed there.

**Comment by pplans** [comment]
Yes if it's not in a nearly stable state perhaps don't waste much of your time yet on this :)

**Comment by parvusdomus** [comment]
no worries, just created a woraround until they fix it and now my dev branch works both in v10 and v11

**Comment by parvusdomus** [comment]
Manually installing v11 works properly with the -> character while updating doesn't. I've sent them the required files to find out the issue and fix it. The bug is totally theirs (in the html sanitization module)

**Comment by parvusdomus** [comment]
Found a deprecation warning while working with translation keys. Will not change until we are actually preparing for v11 (when v11 stable is released) As it is a deprecation warning it will still work until v13
